scripts/install_bb.sh

```diff
@@ -40,7 +40,7 @@ http_download() {
 }
 
 execute() {
-    BB_VERSION="1.0.3"
+    BB_VERSION=$(curl -f https://raw.githubusercontent.com/bytebase/bytebase/main/scripts/VERSION)
```

Comment:
I think it's fine to keep it the current way as we would change all these as part of release.

Reply:
ok, then I will just update the version to the current 1.0.4